build.sbt

```diff
@@ -1,6 +1,6 @@
 name := "spark-avro"
 
-organization := "com.databricks"
+organization := "com.kainos"
 
 scalaVersion := "2.10.5"
```

Comment:
in the travis.yml you reference scala version 2.10.4, here it is 2.10.5. Not sure the mismatch matters, but might want to keep as much consistency as possible here